viam_cartographer.go

```diff
@@ -678,3 +689,17 @@ func CheckQuaternionFromClientAlgo(pose spatialmath.Pose, componentReference str
 	}
 	return nil, "", errors.Errorf("error getting SLAM position: quaternion not given, %v", returnedExt)
 }
+
+// checkCloseAndCloudStatus returns an error if the cartographer service has been previously closed or is being run in the cloud.
+func (cartoSvc *CartographerService) checkCloseAndCloudStatus(cmd string) error {
```

Comment:
```suggestion
func (cartoSvc *CartographerService) isOpenAndRunningLocally(cmd string) error {
```

Reply:
[nit] think a name starting with is might read a little cleaner in all of the functions it is used in

Reply:
done